util/arena/arena.go

```diff
@@ -60,7 +60,7 @@ func NewAllocator(capacity int) *SimpleAllocator {
 // Alloc implements Allocator.AllocBytes interface.
 func (s *SimpleAllocator) Alloc(capacity int) []byte {
 	if s.off+capacity < cap(s.arena) {
-		slice := s.arena[s.off : s.off : s.off+capacity]
```

Comment:
Why change this？
It causes failure to pass the `gofmt`

Reply:
Please update golang to version 1.10, and try to make check again. @wentaoxu

Reply:
ok